snappl/diaobject.py

```diff
@@ -73,19 +73,19 @@ def find_objects( cls, collection=None, subset=None, **kwargs ):
           radius: float, default 1.0
             Radius in arcseconds to search.  Ignored unless ra and dec are given.
 
-          tmax_min, tmax_max: float
-            Only return objects whose tmax is between these limits.
+          mjd_max_min, mjd_max_max: float
```

Comment:
Now that I read `mjd_max_min` and `mjd_max_max`, I wonder if `mjd_peak` might be a better name.  `peak` can then be in the dimension of flux, while `min` and `max` can be in the dimension of time.

Up to you.